code/game/data_huds.dm

```diff
@@ -509,3 +509,40 @@ Diagnostic HUDs!
 	var/image/holder = hud_list[DIAG_AIRLOCK_HUD]
 	holder.icon_state = "electrified"
 	set_hud_image_active(DIAG_AIRLOCK_HUD)
+
+// Species info!
+/mob/living/carbon/human/proc/update_species_hud(species_overide)
+	var/image/holder = hud_list[SPECIES_HUD]
+
+	// to not inder nerf the agent card, make it always show human
+	if(istype(get_idcard(), /obj/item/card/id/advanced/chameleon))
+		holder.icon_state = "species_human"
```

Comment:
I don't understand how this is an ideal path considering not all traitors are human?

Reply:
Shouldn't you be able to customize it or something?